content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,569 @@
+# Running Experiments
+
+## Motivation
+
+Machine Learning and Data Science projects usually involve experimentation.
+Motivation of these experiments can range from finding good hyperparameters to
+checking data and concept drift. DVC 2 introduced a new set of commands to
+manage experiments with minimum boilerplate. It allows to run experiments
+defined by pipelines, track their associated data and model files, set
+parameters for each, push experiment parameters and code to Git remotes without
+committing them, create branches and persist them.
+
+In this part of the DVC User's Guide we will explain how DVC runs the
+experiments, setting parameters, running the experiments in parallel, or in
+temporary directories.
+
+If this is the first time you're introduced to Machine Learning experimentation,
+it may be better to start with the
+[Experiment's Trail](/doc/start/experiments/).
+
+## The Pipeline
+
+DVC uses pipelines composed of <abbr>stages</abbr>. Stages are isolated and
+interdependent steps that has a command, a set of dependencies, and output.
+Stages can depend onto each other and can build a <abbr>DAG</abbr>.
+
+DVC employs the pipeline to run the experiment commands. Each pipeline has an
+end-point, which is the last command in the pipeline. The output of this last
+command is considered the output of experiment. Additionally DVC tracks
+artifacts from intermediate stages, and doesn't reproduce them if their
+dependents have not changed.
+
+DVC pipelines are a detailed topic that we cover in [Get Started][gs-pipelines]
+and the [User's Guide][ug-pipeline-files]. Here we assume that there is already
+a pipeline defined in `dvc.yaml` file in the <abbr>project</abbr>.
+
+[gs-pipelines]: /doc/start/data-pipelines
+[ug-pipeline-files]: /doc/user-guide/project-structure/pipelines-files
+
+### Running the pipeline
+
+The pipeline defined in `dvc.yaml` file is run with default settings using:
+
+```dvc
+$ dvc exp run
```

Comment:
I wonder if we want to avoid the concept of pipeline in this guide actually. We'll probably have to mention and link it (as discussed above) but let's try to keep the language around experimentation? Something like "The current version/variation of your DVC project can be run with `dvc exp run`..."

Reply:
We don't want to avoid the concept of pipelines in the UG. Pipelines are an integral part of experimentation and we would like to present the full details to the user.

Reply:
> We don't want to avoid the concept of pipelines in the UG

Again, we can have a separate pipeline guide in the UG. I'm talking about this one guide which should be about running experiments. I think we have to limit the scope of our guides content or we'll be writing very long tutorials every time which overlap 50% or more with each other.

I'm not so sure that pipelines are an integral part of experimentation, why? You can experiment on a single stage. They're meant as separate features AFAIK.

Reply:
`dvc exp run` is running the pipelines, and I really would like to have experiments without pipelines, but we have this restriction. Which content does this section repeat, apart from `dvc exp run`'s cmd ref?

Reply:
> Which content does this section repeat

It will overlap completely with a future data pipelines guide.

Resolving this for now...